crates/bevy_hierarchy/src/child_builder.rs

```diff
@@ -240,42 +246,45 @@ impl Command for RemoveParent {
     }
 }
 
-/// Struct for building children onto an entity
+/// Struct for building children entities and adding them to a parent entity.
 pub struct ChildBuilder<'w, 's, 'a> {
     commands: &'a mut Commands<'w, 's>,
     push_children: PushChildren,
 }
 
 impl<'w, 's, 'a> ChildBuilder<'w, 's, 'a> {
-    /// Spawns an entity with the given bundle and inserts it into the children defined by the [`ChildBuilder`]
+    /// Spawns an entity with the given bundle and inserts it into the parent entity's [`Children`],
+    /// defined by the [`ChildBuilder`], later adding a [`Parent`] component to the created entity.
```

Comment:
This sentence is not English. This needs a revision. I think the following might help: Usually `ChildBuilder` is a variable named `parent`. Consider that `self` _is_ `parent`.